src/Illuminate/Auth/Middleware/RequirePassword.php

```diff
@@ -63,7 +77,7 @@ public function handle($request, Closure $next, $redirectToRoute = null, $passwo
             }
 
             return $this->responseFactory->redirectGuest(
-                $this->urlGenerator->route($redirectToRoute ?? 'password.confirm')
+                $this->urlGenerator->route($redirectToRoute ?: 'password.confirm')
```

Comment:
`$redirectToRoute` may be `null` if no arguments, but it may also be an empty string when only `passwordTimeoutSeconds` is specified.

```php
"password.confirm:,300"
```